### PR TITLE
refactor: new nlu data table

### PR DESCRIPTION
**Pull request by pheel**
move nlu examples to their own database collection, avoiding mongodb document size limit
improve usability of nlu section with new data table including batch operations and drafting

### CODE REVIEW

**Comment by MatthieuJnon** [comment]
chitchat does not work anymore after migrating my os project

**Comment by MatthieuJnon** [comment]
After adding an entity to an example the mouse is stucked in selection mode, just moving the mouse around selects examples

**Comment by MatthieuJnon** [comment]
switching from chitchat nlu in french to my project (only in english) gives me a grey page, have to reload to fix it

**Comment by MatthieuJnon** [comment]
Also the migration makes me lose 80% of the french examples in chitchat, have you tested the migration with big multi lingual projects ? We have to make extra sure the nlu examples of users are not lost

**Comment by MatthieuJnon** [comment]
https://pastebin.com/n0pxnR4k
Here are the errors I'm seeing on migration, the first one doesn't always pop up

File: botfront/imports/ui/components/nlu/models/Filters.jsx [hunk 1]
```diff
@@ -1,43 +1,105 @@
...
+        const { onChange, filter } = this.props;
+        onChange({ ...filter, sortKey });
+    };
+
+    handleOrderChange = (order) => {
+        const { onChange, filter } = this.props;
+        onChange({ ...filter, order });
+    };
+
+    // handleCalendarChange = (...bounds) => {
```
Comment (MatthieuJnon) [inline botfront/imports/ui/components/nlu/models/Filters.jsx:57]: why is this commented ?

File: botfront/imports/ui/components/nlu/models/Filters.jsx [hunk 2]
```diff
@@ -1,43 +1,105 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Input, Dropdown } from 'semantic-ui-react';
+import {
+    Input, Dropdown, Button, Icon,
+} from 'semantic-ui-react';
+// import moment from 'moment';
```
Comment (MatthieuJnon) [inline botfront/imports/ui/components/nlu/models/Filters.jsx:6]: comments here too

File: botfront/imports/ui/components/nlu/models/Filters.jsx (hunk 1 above)
Comment (pheel) [reply botfront/imports/ui/components/nlu/models/Filters.jsx:57]: these are the lines uncommented on EE

**Pull request merged**